--- a/score/itf/plugins/docker.py
+++ b/score/itf/plugins/docker.py
@@ -331,6 +331,8 @@ def target_init(request, _docker_configuration):
         environment=_docker_configuration["environment"],
         volumes=_docker_configuration["volumes"],
         shm_size=_docker_configuration["shm_size"],
+        privileged=_docker_configuration.get("privileged", False),
+        cap_add=_docker_configuration.get("cap_add"),
     )
     target = None
     try:
